applications/nrf5340_audio/src/modules/audio_sync_timer.c

```diff
@@ -9,91 +9,249 @@
 #include <nrfx_dppi.h>
 #include <nrfx_i2s.h>
 #include <nrfx_ipc.h>
+#include <nrfx_rtc.h>
+#include <nrfx_timer.h>
 
-#include "nrf5340_audio_common.h"
+#include "audio_sync_timer.h"
```

Comment:
This #include "audio_sync_timer.h" should be on top for consistency in the audio application

Reply:
Another nitpick:
Newline after #include "audio_sync_timer.h". This is relevant for audio_sync_timer.c and audio_sync_timer_rtc.c